examples/validate-fee/README.md

```diff
@@ -0,0 +1,28 @@
+# Metagraph - Data Application - Validate Fee
+
+This example demonstrates a implementation of the data application service call _validateFee_
+and the _estimate-fee_ endpoint.
+
+## Template
+
+Primary code for the example can be found in the following files:
+
+`modules/data_l1/src/main/scala/com/my/validate_fee/Main.scala`
+
+## Scripts
+
+`scripts/send_data_transaction.js` is a simple script to send an initial message to the metagraph.
```

Comment:
I don't see a scripts directory here. Does that need to get added to git?

Reply:
For some reason,  the `.gitignore` (copied from the voting-poll example) contains `scripts/`.
@IPadawans Is that by design?

Reply:
You can remove it from gitignore, it should contain the scripts folder. I've already removed it from the new project.

Reply:
the todo example put a .gitignore at the root of the project (since that is where the .git resides) but I'm realizing now that we may want the individual gitignore within each project so euclid to copy?